internal/authority/authority.go

```diff
@@ -34,6 +35,10 @@ const certDuration = 7 * 24 * time.Hour
 //  A T is a TLS certificate authority which can issue client and server
 // certificates and provide configuration for HTTPS clients.
 type T struct {
+	// pem is the PEM-encoded bytes of the CA. The key and certificate are also
+	// stored in other forms for convenient access.
+	pem []byte
```

Comment:
What's the relationship between `pem` and the existing `certPEM` and `keyPEM`? I'm not very familiar with all of this, but it seems like `pem` contains both private key and certificate, which leads me to wonder—is the PEM-encoded form just the (ordered) concatenation of the PEMs of the constituents? I think this comment should describe the relationship, whether it's this or something else.

Reply:
I modified this to just build the contents on the fly. Does this version make the relationship clear?